Formula/xfig.rb

```diff
@@ -0,0 +1,76 @@
+class Xfig < Formula
+  desc "Facility for interactive generation of figures"
+  homepage "https://mcj.sourceforge.io"
+  url "https://downloads.sourceforge.net/mcj/xfig-3.2.8a.tar.xz"
+  sha256 "ba43c0ea85b230d3efa5a951a3239e206d0b033d044c590a56208f875f888578"
+  license "MIT"
+
+  livecheck do
+    url "https://downloads.sourceforge.net/mcj/"
+    regex(/href=.*?xfig[._-](\d+(?:\.\d+)+[a-z]?)\.t/i)
+  end
+
+  # The backend progam, fig2dev, is not needed, but without it, conversion of
+  # fig files to any output format (eps, pdf, tikz, svg, bitmaps) is not
+  # possible. Hence, xfig without fig2dev is not overly useful. In addition,
+  # users experience missing capabilites of xfig and do not want to investigate
+  # the specific requirements for their use case.
+  depends_on "fig2dev"
+
+  # The minimum dependencies are really
+  #   libx11, libxaw || libxaw3d, libxi, libxt.
+  # All others are, strictly speaking, ":recommended". However, without
+  # ghostscript and the image libraries, display of imported images on the xfig
+  # drawing canvas and, partially, determination of the original size of the
+  # images, is not possible. brew audit complained about the ":recommended" tag
+  # for core Formulae, so make these image libraries hard dependencies.
```

Comment:
```suggestion
  depends_on "fig2dev"
```
Let's cut these comments out. I don't think they're needed in the formula. You can write them here in your pull request where someone can look them up if need be.